apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/BlockchainProcessorImpl.java

```diff
@@ -945,6 +950,7 @@ public void run() {
         }
     };
 
+    @Inject
     private BlockchainProcessorImpl(BlockValidator validator) {
```

Comment:
Does it inject validator correctly?

Reply:
Yes, it does now, but can break potentially and some point.